### PR TITLE
Refactor components

### PR DESCRIPTION
**Pull request by lucas-gauer**
- Rewrite `Buttons` body for better reading
- Avoid returning null on `Fullscreen` component
- Move `AccountMenu` out of `AccountHeader`
- Create a `LogoHeader` base on `Section` and split the headers (from `NormalHeader` and `Header`)
- Rename `Layout` -> `OnboardingLayout`
- Rename `AuthViewLayout` -> `Layout`
- Remove `Link` component (use Link from react-router-dom instead)
- Move TxLayouts to Layouts folder (from layout/tx)
- Remove unused parameters and avoid duplicated code on `Modal`
- Rewrite `Card` to avoid duplicated code. Create `SimpleCard`, `ErrorCard` and `InfoCard`

### CODE REVIEW

File: source/components/Button/Button.tsx [hunk 1]
```diff
@@ -50,20 +50,30 @@ export const Button = ({
 const disabledStyle =
   'text-button-disabled cursor-not-allowed font-light border-2 border-button-disabled';
 
-export const PrimaryButton: FC<IPrimaryButton> = ({
+export const PrimaryButton: React.FC<IPrimaryButton> = ({
+  action = false,
   children,
   disabled = false,
+  id = '',
```
Comment (lucas-gauer) [inline source/components/Button/Button.tsx:57]:
```suggestion
  id = '0',
```

File: source/components/Layout/Layout.tsx [hunk 2]
```diff
@@ -1,50 +1,66 @@
...
+  id = '',
+  title,
+}) => {
+  const navigate = useNavigate();
+  const { browser } = useBrowser();
+
+  const url = browser.runtime.getURL('app.html');
+
+  return (
+    <div className="relative w-full min-w-popup h-full min-h-popup text-brand-white bg-bkg-2">
```
Comment (amandagonsalves) [inline source/components/Layout/Layout.tsx:25]: layouts are breaking in fullscreen view

File: source/components/Layout/TxLayout.tsx [hunk 3]
```diff
@@ -22,50 +22,45 @@ interface ITxLayout {
 }
 
 export const TxLayout: FC<ITxLayout> = ({ confirmRoute, txType, title }) => {
-  const controller = useController();
+  const accountCtlr = useController().wallet.account;
```
Comment (amandagonsalves) [inline source/components/Layout/TxLayout.tsx:25]:
```suggestion
  const accountController = useController().wallet.account;
```

we don't need to abbreviate variable names, it is more difficult to read

File: source/components/Layout/TxLayout.tsx [hunk 4]
```diff
@@ -22,50 +22,45 @@ interface ITxLayout {
...
+  const transaction = accountCtlr.getTemporaryTransaction(txType);
```
Comment (amandagonsalves) [inline source/components/Layout/TxLayout.tsx:26]:
```suggestion
  const transaction = accountController.getTemporaryTransaction(txType);
```

File: source/components/Layout/TxLayout.tsx [hunk 5]
```diff
@@ -22,50 +22,45 @@ interface ITxLayout {
...
   useEffect(() => {
-    handleGetFee();
-  }, []);
+    getFee();
+  });
 
-  const handleCreateTemporaryTransaction = ({ fee }) => {
-    controller.wallet.account.updateTemporaryTransaction({
+  const updateTemporaryTransaction = ({ fee }) => {
+    accountCtlr.updateTemporaryTransaction({
```
Comment (amandagonsalves) [inline source/components/Layout/TxLayout.tsx:48]:
```suggestion
    accountController.updateTemporaryTransaction({
```

**Review by amandagonsalves — CHANGES_REQUESTED** [review]

File: source/components/Layout/TxLayout.tsx (hunk 5 above)
Comment (amandagonsalves) [reply source/components/Layout/TxLayout.tsx:48]: there are also some conflicts to resolve before merge

**Review by amandagonsalves — APPROVED** [review]

**Pull request merged**